openmdao/docs/features/experimental/xdsm_visualization.rst

```diff
@@ -0,0 +1,113 @@
+.. _xdsm_generation:
+
+***************
+XDSM generation
+***************
+
+OpenMDAO also supports the generation of XDSM diagrams of models. For more information about XDSM diagrams see XDSM_Overview_.
+
+.. _XDSM_Overview: http://mdolab.engin.umich.edu/content/xdsm-overview
+
+The diagrams can be in the form of HTML or LaTeX files.
+
+To use this feature the you need to install PyXDSM. This can be done by:
+
+.. code-block:: none
+
+    git clone https://github.com/mdolab/pyXDSM
+    cd pyXDSM
+    python setup.py install
+
+To use the feature which generates LaTeX files, LaTeX must be installed.
+
+You can generate XDSM files either from the command line or from a script.
+
+From the Command Line
+---------------------
+
+.. _om-command-view_xdsm:
+
+Generating an XDSM diagram for a model from the command line is easy. First, you need a Python script that contains
+and runs the model.
+
+.. note::
+
+    To get into more details, if :code:`final_setup` isn't called in the script (either directly or as a result of :code:`run_model`
```

Comment:
could mention here that when using the command line version, even if the script does run_model or run_driver, the script will terminate after final_setup and will not actually run the model.

Reply:
done. Thanks